### ISSUE TITLE
Firefox react error

### ISSUE THREAD

**Issue by arikan** [polkadot-js/apps#373]
Firefox Developer Edition 64.01 throws a react error

https://polkadot.js.org/apps/#/explorer

<img width="1446" alt="firefox developer editionscreensnapz139" src="https://user-images.githubusercontent.com/6969/47430908-ab946580-d768-11e8-8f29-f5c832390d00.png">

**Comment by jacogr** [comment]
Yes, sadly https://github.com/polkadot-js/apps/issues/132 - nothing much we can do atm when connecting to a local node. (Potentially should actually just warn when in FF and connecting to a local node)
